tools/explorer/lib/ontology_wrapper.py

```diff
@@ -61,16 +59,16 @@ def GetFieldsForTypeName(
     """Gets a list of fields for a given typename within a namespace.
 
     Args:
-       namespace: the name of the namespace as a string.
-       entity_type_name: the name of the entity type as a string.
-       required_only: when true will return only required fields for a given
-         type.
+      namespace: the name of the namespace as a string.
+      entity_type_name: the name of the entity type as a string.
+      required_only: when true will return only required fields for a given
+        type.
 
     Returns:
```

Comment:
is the shift in indentations on purpose?

Reply:
For some reason, everything was indented by 3 spaces instead of 2, so I fixed it to make the indentation uniform across the whole library.